src/components/Channels.js

```diff
@@ -1,25 +1,35 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux';
 import { fetchChannels } from '../actions/channelActions';
-
+import Card from 'react-bootstrap/Card'
 class Channels extends Component {
   componentDidMount(){
     this.props.fetchChannels();
   }
 
   render() {
+    
     const channelItems = this.props.channels.map(channel => (
-      <div key={channel.name}>
-        <h2> {channel.name} </h2>
-        <p> {channel.category} </p>
-      </div>
+      <Card key={channel.name} style={{ width: '18rem' }}>
+        <Card.Body>
+          <Card.Title>{channel.name}</Card.Title>
+          <Card.Subtitle className="mb-2 text-muted">{channel.category}</Card.Subtitle>
```

Comment:
Not sure if it's really worth it to display the channel's category on each channel